incubator/kafqa/values.yaml

```diff
@@ -0,0 +1,73 @@
+# Default values for kafqa.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+environment: "integration"
+appDurationMs: "5000000000"
+acks: "1"
+appPort: 8081
+namespace: "hermes"
+
+kafka:
+  cluster: "test"
+  topic: "kafqa-test-topic"
+
+producer:
```

Comment:
by default enable producer & consumer, and make that as default value in template itself.

Reply:
Yes we can enable. But what is the benefit of having the default value in the template ?